build-and-test.ps1

```diff
@@ -40,6 +42,7 @@ else {
 $manifest = Get-Content "manifest.json" | ConvertFrom-Json
 $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
+Invoke-CleanupDocker $activeOS
```

Comment:
Consider moving this logic along with the logic to retrieve the activeOS before the manifest is loaded.

Reply:
Ok. Say it is moved to line 34, then the code look like this -

```powershell
$(docker version) | % { Write-Host "$_" }
$activeOS = docker version -f "{{ .Server.Os }}"
Invoke-CleanupDocker $activeOS
```

`docker version` getting called in subsequent statements makes me uncomfortable. These two calls were present prior to this PR, but makes it more glaring now.